Open Judge System/Web/OJS.Web/Scripts/Administration/Problems/problems-edit.js

```diff
@@ -44,17 +67,30 @@
             numericTextBox.enable(true);
             input.attr("data-val-required", "Лимита е задължителен!");
 
-            $("form").removeData("validator");
-            $("form").removeData("unobtrusiveValidation");
-            $.validator.unobtrusive.parse($('form'));
+            $("#edit-form form").removeData("validator");
+            $("#edit-form form").removeData("unobtrusiveValidation");
+            $.validator.unobtrusive.parse($('#edit-form form'));
         }
         else {
             numericTextBox.enable(false);
             input.removeAttr("data-val-required");
 
-            $("form").removeData("validator");
-            $("form").removeData("unobtrusiveValidation");
-            $.validator.unobtrusive.parse($('form'));
+            $("#edit-form form").removeData("validator");
+            $("#edit-form form").removeData("unobtrusiveValidation");
+            $.validator.unobtrusive.parse($('#edit-form form'));
+        }
+    });
+
+    $("#edit-form form").on("submit", function (ev) {
+        var submissionTypes = $('input[name^="SubmissionTypes"][name$=".IsChecked"]');
```

Comment:
This code also looks very familiar to me. 
Please try to avoid copy/paste. Better extract in generic function with additional parameters